src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_NUMBER = 100;
+    private Random random = new Random();
```

Comment:
The `random` field should be a constant. According to the Java naming conventions, constants should be in uppercase. Please rename it to `RANDOM`.